roles/piper/tasks/main.yml

```diff
@@ -78,5 +78,6 @@
   template: src="uppmax_global_config.xml.j2" dest="{{ ngi_piper_bin_dest }}/workflows/uppmax_global_config.xml"
   tags: ngi_piper 
 
-
+- name: deploy script (gen_GATK_ref.sh) for manual GATK reference generation
+  template: src="gen_GATK_ref.sh" dest="{{ ngi_piper_bin_dest }}/gen_GATK_ref.sh"
```

Comment:
Add a `tags: ngi_piper` here, so that the playbook can be run with the `--tags ngi_pipeline` argument, if an operator doesn't want to run all the roles in the playbook.